app/core/models.py

```diff
@@ -24,3 +26,39 @@ def save(self, *args, **kwargs):
         if not self.pk and XMSConfiguration.objects.exists():
             raise ValidationError('XMSConfiguration model already exists')
         return super(XMSConfiguration, self).save(*args, **kwargs)
+
+
+class ReceiverEmailConfiguration(models.Model):
+    """Model for Email Configuration """
+
+    email_address = models.EmailField(
+        max_length=254,
+        help_text='Enter email personas addresses to send log data',
+        unique=True)
+
+    def get_absolute_url(self):
+        """ URL for displaying individual model records."""
+        return reverse('Configuration-detail', args=[str(self.id)])
+
+    def __str__(self):
+        """String for representing the Model object."""
+        return f'{self.id}'
+
+    def save(self, *args, **kwargs):
+        email_verification(self.email_address)
+        return super(ReceiverEmailConfiguration, self).save(*args, **kwargs)
+
+
+class SenderEmailConfiguration(models.Model):
+    """Model for Email Configuration """
+
+    sender_email_address = models.EmailField(
+        max_length=254,
+        help_text='Enter sender email address to send log data from',
+        default='openlxphost@gmail.com')

```

Comment:
We should probably refrain from using a Gmail address here, even if we think it's not a real one -- you just never know.